applications/ConvectionDiffusionApplication/python_scripts/convection_diffusion_stationary_embedded_solver.py

```diff
@@ -22,11 +22,17 @@ def __init__(self, main_model_part, custom_settings):
         # Construct the base solver and validate the remaining settings in the base class
         super().__init__(main_model_part, custom_settings)
 
+        # Overwrite the base solver minimum buffer size
+        self.min_buffer_size = 1
+
         KratosMultiphysics.Logger.PrintInfo(self.__class__.__name__, "Construction finished")
 
     @classmethod
     def GetDefaultParameters(cls):
         this_defaults = KratosMultiphysics.Parameters(r"""{
+            "mls_extension_operator_order" : 1,
+            "use_mls_constraints" : true,
```

Comment:
```suggestion
            "use_mls_constraints" : false,
```
I'd set to `false` by default to avoid modifying current behavior.